scripts/create-release-branch.js

```diff
@@ -1,28 +1,38 @@
 #!/usr/bin/env node
 
 const shell = require('shelljs');
-const { checkVSCodeVersion, checkLernaInstall, checkBaseBranch } = require('./validation-utils');
+const {
+  checkVSCodeVersion,
+  checkLernaInstall,
+  checkBaseBranch
+} = require('./validation-utils');
 const logger = require('./logger-util');
 
 shell.set('-e');
 shell.set('+v');
 
+const currentVersion = require('../packages/salesforcedx-vscode/package.json')
```

Comment:
Had to make some changes in order to dynamically get the weekly version number. So we'll get the current version from one of the package.json's, bump the minor version and then set it as `SALESFORCEDX_VSCODE_VERSION`. The rest is the same logic as before.

Reply:
I think we should modify this logic so that if the SALESFORCEDX_VSCODE_VERSION env has not already been set, we will fire this logic. Otherwise this prevents a user from overriding it without making script changes.

Reply:
Also, how frequently do we want to bump the major version?

Reply:
> Otherwise this prevents a user from overriding it without making script changes

I don't want a user to override this value, at least not when running the build via CircleCI. 

> Also, how frequently do we want to bump the major version?

That will keep happening on every mayor platform release we'll update the mayor version (3 times a year).

Reply:
Is there a way to pass in arguments to this script instead of env setting? If so, that will be even better so we don't end up reusing an old env setting.

Reply:
@jag-sfdc yes, we could have this code for extracting the value from a package.json as part of the circleci step in the config file that calls this script. I decided to have it here since it's easier to test outside of circleci.

The other option would be to have the value as a real build environment variable but that would mean someone would have to manually change that config on CircleCI everytime we decide to release (at least once a week).